users/config.rst

```diff
@@ -341,6 +349,99 @@ markerName
+        sequence at random. Provides acceptable data distribution, minimal spinning disk strain.
+
+    random:
+        File parts are downloaded in a random order. Provides great data distribution, but very taxing on
+        spinning disk drives
+
+    inOrder:
+        File parts are downloaded sequentially, from start to finish. Spinning disk drive friendly, but provides
+        no improvements to data distribution.
+
+copyRangeMethod
+    Provides an choice of method for copying data between files. This can be used to optimise copies on network
+    filesystems, improve speed of large copies or clone the data using copy-on-write functionality if the underlying
+    filesystem supports it.
+
+    .. warning::
+        This is an experimental feature, so please use at your own risk.
+
+    standard (default)
+        Reads the data from source file into application memory, writes the data from application memory into the
+        destination file.
+
+        Available on: All platforms
+
+    copy_file_range
+        Uses copy_file_range syscall, which if underlying filesystem supports, uses copy-on-write semantics to
+        clone the data. Introduced in Linux 4.5 and tested on XFS and BTRFS. Some network filesystems might use this
+        to perform server-side copies.
+
+        Tested on: BTRFS, XFS
+        Available on: Linux
+
+    ioctl
+        Uses ioctl syscall with FICLONERANGE option, which if underlying filesystem supports, uses copy-on-write
+        semantics to clone the data. Officially introduced in Linux 4.5, but was previously known as
+        BTRFS_IOC_CLONE_RANGE, which was used to provide copy-on-write semantics to BTRFS filesystems since Linux 2.6.29.
+        Some network filesystems might use this to perform server-side copies. Will fail if not supported by the
+        underlying filesystem.
+
+        Tested on: BTRFS
+        Not available on: Windows, Solaris, Darwin (OS X)
+
+    sendfile
+        Uses sendfile syscall, which performs in-kernel copy, avoiding having to copy the data into application memory.
+
+        Tested on: BTRFS, XFS
+        Not available on: Windows, Darwin (OS X)
+
+    duplicate_extents
+        Uses Windows Block Cloning via FSCTL_DUPLICATE_EXTENTS_TO_FILE, which provides copy-on-write semantics to clone
+        the data. Requires Windows Server 2016 or later, and a compatible filesystem (ReFS, SMB 3.1.1, CsvFS). Will
+        fail if not supported by the underlying filesystem.
+
+        .. warning::
+            Completely untested, use at your own risk.
+
+        Available on: Windows
+
+    all
+        Tries all of the copy methods in the following order: ioctl, copy_file_range, sendfile, duplicate_extents,
+        standard.
+
+        Available on: All platforms
+
+        .. warning::
+            Not recommended on Windows as it has not been tested, might be very costly on all other platforms.
```

Comment:
This could be read as implying it's been tested, regardless of the initial warning. I'd reword to

```suggestion
            Not recommended as it has not been tested. Also might be very costly on all platforms but Windows.
```

Reply:
This was tested on non-windows to some extent.